index.bs

```diff
@@ -3813,6 +3828,55 @@ ScriptEvent = (
 )
 </pre>
 
+### Preload Scripts ### {#preload-scripts}
+
+A <dfn>Preload script</dfn> is one which runs on creation of a new {{Window}},
+before any author-defined script have run.
+
+TODO: Extend this to scripts in other kinds of realms.
+
+A [=BiDi session=] has a <dfn>preload script map</dfn> which is a map in which the
+keys are [[!RFC4122|UUID]]s, and the values are [=structs=] with an <a
+for=struct>item</a> named <code>expression</code>, which is a string, and an
+item named <code>sandbox</code> which is a string or null.
+
+<div algorithm>
+To <dfn export>run WebDriver BiDi preload scripts</dfn> given |environment settings|:
```

Comment:
Do we have to specify when this algo should be triggered?

Reply:
That's specified in HTML: https://github.com/whatwg/html/pull/8300